generator/generator_test.cc

```diff
@@ -141,7 +141,7 @@ TEST_F(GeneratorTest, BadCommandLineArgs) {
 }
 
 TEST_F(GeneratorTest, GenerateServicesSuccess) {
-  int constexpr kNumMockOutputStreams = 23;
+  int constexpr kNumMockOutputStreams = 25;
```

Comment:
That looks unfortunate.

Reply:
Eh, it's not so bad. I'd rather have the test than not.